src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -0,0 +1,406 @@
+   */
+  @SerializedName("billing")
+  String billing;
+
+  /**
+   * Define thresholds at which an invoice will be sent, and the subscription advanced to a new
+   * billing period.
+   */
+  @SerializedName("billing_thresholds")
+  Subscription.BillingThresholds billingThresholds;
+
+  /**
+   * Time at which the subscription schedule was canceled. Measured in seconds since the Unix epoch.
+   */
+  @SerializedName("canceled_at")
+  Long canceledAt;
+
+  /**
+   * Time at which the subscription schedule was completed. Measured in seconds since the Unix
+   * epoch.
+   */
+  @SerializedName("completed_at")
+  Long completedAt;
+
+  /** Time at which the object was created. Measured in seconds since the Unix epoch. */
+  @SerializedName("created")
+  Long created;
+
+  /**
+   * Object representing the start and end dates for the current phase of the subscription schedule,
+   * if it is `active`.
+   */
+  @SerializedName("current_phase")
+  CurrentPhase currentPhase;
+
+  /** ID of the customer who owns the subscription schedule. */
+  @SerializedName("customer")
+  @Getter(lombok.AccessLevel.NONE)
+  @Setter(lombok.AccessLevel.NONE)
+  ExpandableField<Customer> customer;
+
+  /** Unique identifier for the object. */
+  @Getter(onMethod = @__({@Override}))
+  @SerializedName("id")
+  String id;
+
+  /** The subscription schedule's default invoice settings. */
+  @SerializedName("invoice_settings")
+  SubscriptionScheduleInvoiceSettings invoiceSettings;
+
+  /**
+   * Has the value `true` if the object exists in live mode or the value `false` if the object
+   * exists in test mode.
+   */
+  @SerializedName("livemode")
+  Boolean livemode;
+
+  /**
+   * Set of key-value pairs that you can attach to an object. This can be useful for storing
+   * additional information about the object in a structured format.
+   */
+  @Getter(onMethod = @__({@Override}))
+  @SerializedName("metadata")
+  Map<String, String> metadata;
+
+  /** String representing the object's type. Objects of the same type share the same value. */
+  @SerializedName("object")
+  String object;
+
+  /** Configuration for the subscription schedule's phases. */
+  @SerializedName("phases")
+  List<Phase> phases;
+
+  /**
+   * Time at which the subscription schedule was released. Measured in seconds since the Unix epoch.
+   */
+  @SerializedName("released_at")
+  Long releasedAt;
+
+  /** ID of the subscription once managed by the subscription schedule (if it is released). */
+  @SerializedName("released_subscription")
+  String releasedSubscription;
+
+  /** Behavior of the subscription schedule and underlying subscription when it ends. */
+  @SerializedName("renewal_behavior")
+  String renewalBehavior;
+
+  /**
+   * Interval and duration at which the subscription schedule renews for when it ends if
+   * `renewal_behavior` is `renew`.
+   */
+  @SerializedName("renewal_interval")
+  SubscriptionScheduleRevision.RenewalInterval renewalInterval;
```

Comment:
I think this is backwards. IMO the class should be in `SubscriptionSchedule` and `SubscriptionScheduleRevision` should do this instead:
```
SubscriptionSchedule.RenewalInterval renewalInterval;
```

Reply:
Now nested in `SubscriptionSchedule`